SetMeta/Impl/OptionSetParserV1.cs

```diff
@@ -38,22 +40,54 @@ public override OptionSet Parse(XmlTextReader reader, IOptionSetValidator option
             if (body == null)
                 throw new InvalidOperationException("Xml body is absent.");
 
-            optionSet.Version = Version;
+            optionSet.Version = Version;            
 
             ParseConstants(body, optionSet.Constants);
+
+            body = CheckForConstantsValue(body, optionSet);
+
             ParseOptions(body, optionSet.Options);
             ParseGroups(body, optionSet.Groups);
 
             return optionSet;
         }
 
+        private XElement CheckForConstantsValue(XElement body, OptionSet optionSet)
```

Comment:
No, we not need to parse whole xml at once, we must replace data only at expected places